src/Ansatz.jl

```diff
@@ -388,6 +389,50 @@ function expect(ψ::AbstractAnsatz, observables::AbstractVecOrTuple; bra=adjoint
     end
 end
 
+function expect(::NonCanonical, ψ::AbstractAnsatz, observable; bra=adjoint(ψ))
+    return contract(merge(ψ, observable, bra))
+end
+
+# TODO: Try to find a better way to do this
+function expect(::MixedCanonical, ψ::AbstractAnsatz, observable; bra=adjoint(ψ))
+    return contract(merge(ψ, observable, bra))
```

Comment:
the way to do this is to move the orthogonality center to the site(s) of the observable and then just contract the tensors on the orthogonality center with the observable

but yeah, we can implement it later

do you mind opening an issue so we don't forget?